modules/ROOT/assets/attachments/sync-gateway-public.yaml

```diff
@@ -164,7 +164,23 @@ parameters:
   heartbeat:
     in: query
     name: heartbeat
-    description: Default is 0. Interval in milliseconds at which an empty line (CRLF) is written to the response. This helps prevent gateways from deciding the socket is idle and closing it. Only applicable to longpoll or continuous feeds. Overrides any timeout to keep the feed alive indefinitely. Setting to 0 results in no heartbeat.
+    description: |+
+      The `heartbeat` defines the interval (in milliseconds) at which an empty line (CRLF) is written to the response. It helps prevent gateways from deciding the socket is idle and closing it.
+
+      The `heartbeat` value overrides any `timeout` value, to keep the feed alive indefinitely.
+
+      Setting `heartbeat=0` results in no heartbeat.
+
+      **Default:** 0, which is no heartbeat
+
+      **Constraints:**
+
+      - Applies ONLY where `feed=longpoll` or `feed=continuous`.
+
+      - Minimum: 25 seconds
```

Comment:
Since the description above says the value is in milliseconds, you should put all values as milliseconds - you can add a more "conventional" units in parens.

EX: 25,000 milliseconds (25 seconds)

Reply:
Makes sense ... applied in e5adab4